### PR TITLE
Add a first pass mlflow v2 end to end example

### LINKED ISSUE #3479: Create an end-2-end notebook example for v2 mlflow model serving

### PR DESCRIPTION
**Pull request by sakoush**
**What this PR does / why we need it**:
Adds a notebook to showcase building a model using mlflow, pack and deploy in seldon-core (local kind cluster)

**Which issue(s) this PR fixes**:

Fixes #3479

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
```

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **axsaucedo**
You can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: examples/models/mlflow_v2_protocol_end_to_end/mlflow_v2_protocol_end_to_end.ipynb
Comment (axsaucedo) [inline examples/models/mlflow_v2_protocol_end_to_end/mlflow_v2_protocol_end_to_end.ipynb:1]: <p>Probably best to clear this cell if left here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3483/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (axsaucedo) [inline examples/models/mlflow_v2_protocol_end_to_end/mlflow_v2_protocol_end_to_end.ipynb:1]: <p>Would be good to add subtitels to these sections as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3483/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

**Review by axsaucedo — CHANGES_REQUESTED** [review]
Added a couple of comments - also a couple of extra things required to move around:
* Rename notebook to README.pynb
* Convert readme to markdown so people accessing from github can read it as well

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[axsaucedo](https://github.com/SeldonIO/seldon-core/pull/3483#issuecomment-897660341 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
